_layouts/base.html.haml

```diff
@@ -140,31 +140,66 @@
       .container#Content(style="padding-top: 20px;")
         ~ content
 
-    %footer#footer
+    %footer
       .container
         .row
-          .col-md-6
-            %p
-              Development is sponsored by
-              %a(href="http://www.redhat.com/")<
+          .col-md-3
+            %h4 KIE projects
+            %ul
+              %li<
+                %a(href="https://www.drools.org/" target="_blank") Drools rule engine
+              %li<
+                %a(href="https://www.optaplanner.org/" target="_blank") OptaPlanner constraint solver
+              %li<
+                %a(href="https://www.jbpm.org/" target="_blank") jBPM workflow engine
+          .col-md-3
+            %h4 Community
+            %ul
+              %li<
+                %a(href="http://kverlaen.blogspot.com" target="_blank") Blog
+              %li<
+                %a{:href => relative("/community/getHelp.html")} Get Help
+              %li<
+                %a{:href => relative("/community/team.html")} Team
+              %li<
+                %a{:href => relative("/community/governance.html")} Governance
+              %li<
+                %a{:href => relative("/community/research.html")} Academic research
+          .col-md-3
+            %h4 Code
+            %ul
+              %li<
+                %a{:href => relative("/code/sourceCode.html")} Build from source
+              %li<
+                %a{:href => site.issueTracker} Submit a bug
+              %li<
+                %a{:href => relative("/code/license.html")} License (Apache)
+              %li<
+                %a{:href => site.pom.latest.releaseNotesVersion} Release notes
+              %li<
+                %a{:href => relative("/download/upgradeRecipe/upgradeRecipe7.0.html")} Upgrade recipes
```

Comment:
don't think jbpm has any update recipe

Reply:
the page is there - but if you prefer i can take out this link too

Reply:
I don't see it anywhere on current website so I'd opt at removing it